### DISCUSSION TITLE
Support for ZeroMQ

### DISCUSSION THREAD

**Discussion by divaltor** [Lancetnik/Propan#35] [Ideas]
Hello, thank you so mush for this library. Want to ask is it possibly to add support for ZeroMQ broker?

https://zeromq.org/

**Comment by Lancetnik** [comment]
Hello! Sure, I am trying to implement as many MQ Broker supporting as possible. But, ZeroMQ is not in my priority right now: I am working on SQS, Kafka RPC, Redis Streams.
But, u can implement ZeroMQ broker by yourself: it's not a big deal, [check this doc article](https://lancetnik.github.io/Propan/contributing/4_adapters/). Happy to merge your PR this case.
Also, my priorities are able to change, if somebody else supports your issue.

**Comment by Lancetnik** [comment]
Well, it should be interesting: ZeroMQ not require any external dependencies. People asks me about lightweight Pub/Sub like this. I had thinking about Websockets/MQTT/GRPC, but ZeroMQ should be best choice right now.
